include/beast/http/basic_parser_v1.hpp

```diff
@@ -280,12 +281,12 @@ class basic_parser_v1 : public detail::parser_base
     std::uint64_t content_length_;
     pmf_t cb_;
     state s_              : 8;
-    unsigned flags_       : 8;
     unsigned fs_          : 8;
     unsigned pos_         : 8; // position in field state
     unsigned http_major_  : 16;
     unsigned http_minor_  : 16;
     unsigned status_code_ : 16;
+    unsigned flags_ = 0;
```

Comment:
Probably want to declare this `uint16_t`. We don't need the whole width of an int.